parameter/CMakeLists.txt

```diff
@@ -107,11 +104,13 @@ include_directories(
     "${PROJECT_SOURCE_DIR}/xmlserializer"
     "${PROJECT_SOURCE_DIR}/utility"
     "${PROJECT_SOURCE_DIR}/remote-processor"
-    "${PROJECT_SOURCE_DIR}/parameter/log/include")
+    "${PROJECT_SOURCE_DIR}/parameter/"
+    "${PROJECT_SOURCE_DIR}/parameter/log/include"
+    "${PROJECT_SOURCE_DIR}/parameter/criterion/include")
 
 # No need to link with libremote-processor: it is accessed via dlopen()
 find_library(dl dl)
-target_link_libraries(parameter xmlserializer pfw_utility dl)
+target_link_libraries(parameter xmlserializer pfw_utility criterion)
```

Comment:
You probably didn't mean to removing the linking to libdl. see 7f7a206ed4f765e80c24fa88ddfea24a4bbb9442